generators/client/templates/react/src/main/webapp/app/shared/util/entity-utils.ts.ejs

```diff
@@ -38,7 +38,12 @@ export const cleanEntity = entity => {
  * @param idList Elements to map.
  * @returns The list of objects with mapped ids.
  */
- export const mapIdList = (idList: ReadonlyArray<any>) => idList.filter((entityId: any) => entityId !== '').map((entityId: any) => ({ id: entityId }));
+export const mapIdList = (idList: ReadonlyArray<any>) => {
```

Comment:
indent

Reply:
all the files have this indentation in `react/src/app`

Reply:
oh ok, the bad indent is the following function :D nsot the part you modified